source/scripts/app.js

```diff
@@ -9,10 +9,43 @@ const DEFAULT_READY_TIME = 300;
 let COOKBOOK_TO_EDIT = null;
 const DEFAULT_COOKBOOK_NAME = "My cookbook";
 
-const router = new Router("home-page");
+const router = new Router("home-page", "home-page");
 const spoonacular = new SpoonacularInterface();
 const indexedDb = new IndexedDbInterface();
 
+/**
+ * Attaches "click" event listeners to the back button on the recipe page
+ * that return to the previous page when clicked.
+ */
+function connectRecipeBackButton() {
+  "use strict";
+  //Get references to buttons in shadowRoot
+  let recipePage = document.querySelector("recipe-page");
+  let recipeShadow = recipePage.shadowRoot;
+  let backButtonRecipe = recipeShadow.getElementById("back-button");
+
+  backButtonRecipe.addEventListener("click", () => {
+    router.return();
```

Comment:
We can use the getter here instead
```suggestion
    let prevPage = router.prevPage;
    router.navigate(prevPage);
```